shared/src/test/scala/scala/util/parsing/combinator/lexical/StdLexicalTest.scala

```diff
@@ -0,0 +1,142 @@
+package scala.util.parsing.combinator.lexical
+
+import org.junit.Test
+import org.junit.Assert.assertEquals
+
+import scala.util.parsing.input.Reader
+
+import scala.collection.mutable.ListBuffer
+import java.awt.RenderingHints.Key
```

Comment:
I think this import isn't used.

Reply:
@Philippus you are absolutely correct. Sorry about that. The import actually made it's way into this file in the previous pull request (PR402). This PR only makes a few changes to this file.
```
```
However, because branch issue-399 currently is on top of issue-397 you see the diff of both issues in this PR. I will remove the unused import and rebase this PR on top of 2.x.